go.mod

```diff
@@ -44,7 +44,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.1
 	github.com/takuoki/gocase v1.0.0
-	github.com/tendermint/flutter/v2 v2.0.3
+	github.com/tendermint/flutter/v2 v2.0.4
```

Comment:
The `go.sum` is missing the entry for "github.com/tendermint/flutter/v2 v2.0.4". This might happen because the version was not updated using something like:
``` 
go get github.com/tendermint/flutter/v2@V2.0.4
```
The command will also update the `go.sum` to match the new version.

Also note that the "v2.0.4" release is using and uppercase "V" in the command because the tag name in "github.com/tendermint/flutter" is "V2.0.4". Ideally the tag should be renamed to "v2.0.4" and then you could run the command using a lower case "v" for the version.

Reply:
done by [e60ee84](https://github.com/ignite-hq/cli/pull/2475/commits/e60ee847eaf399b99ae59a6ee0d40e170d4b7a0f).